client/my-sites/theme/main.jsx

```diff
@@ -387,12 +353,57 @@ const ThemeSheet = React.createClass( {
 	},
 } );
 
+const WrappedThemeSheet = ( props ) => {
+	if ( ! props.isLoggedIn || props.selectedSite ) {
```

Comment:
I considered turning `<ThemesSiteSelectorModal/>` into a more multi-purpose `<ThemesActionBinder/>` that would bind single-site actions to `site` and just pass through its children when logged-out, but that seemed to dilute concerns. I'd rather let consumers of the component handle if they use it or not. Ideally tho this could be expressed more tersely, by a ternary and some sort of identity/pass-thru component...

I guess I'd rather leave as-is for now :-)